src/System.Reflection.Primitives/ref/System.Reflection.Primitives.cs

```diff
@@ -294,6 +296,7 @@ public enum OperandType
         InlineI8 = 3,
         InlineMethod = 4,
         InlineNone = 5,
+        InlinePhi = 6,
```

Comment:
Obsolete attributes need to be in the ref, if they were in standard. Eg in standard I see

```
        [System.ObsoleteAttribute("This API has been deprecated. http://go.microsoft.com/fwlink/?linkid=14202")]
        InlinePhi = 6,
```